.github/workflows/test-dev.yml

```
@@ -20,16 +20,122 @@ permissions:
+      - name: Check for changes that require the container to be rebuilt
+        uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            devcontainer:
+              - .devcontainer/*
+
+      - name: On 'devel' and the devcontainer needs to be rebuilt
+        id: tag
+        # if: steps.changes.outputs.devcontainer == 'true' && steps.extract_branch.outputs.branch == 'devel'
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:32: Double quote to prevent globbing and word splitting

(shellcheck)